src/hooks/useForm.js

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+
+export default function useForm(initialValues){
+    const [values, setValues] = useState(initialValues);
+
+    const handleChanges = e => {
+        e.persist();
+
+        setValues({
+            ...values,
+            [e.target.name]: e.target.value
+        });
+
+        console.log(values);
+    };
+
+    const handleSubmit = e => {
```

Comment:
This should also take a `onSubmit()` function as an argument so that you can let the hook know what you'd like the hook to do when the form is submitted. If you don't feel that's the hooks responsibility, then I would remove it from here

Reply:
I don't think it should be the responsibility of the hook to handle the submit. My mistake. I will get that removed.